tests/requirements.txt

```diff
@@ -1,2 +1,3 @@
 nose
 pycodestyle
+mockldap
```

Comment:
Is there a reason behind the order here? If not, could we make it lexicographical, so `mockldap` is the first?